fission-core/src/capabilities/fission.rs

```diff
@@ -211,65 +214,166 @@ mod tests {
+
+        let server_sk = ed25519_dalek::SigningKey::generate(&mut thread_rng());
+        let server_signer =
+            ucan::did::preset::Signer::Key(ucan::did::key::Signer::EdDsa(server_sk));
+        let server_ed_did_key = EdDidKey::new(server_sk);
+
+        let server = ucan::did::preset::Verifier::Key(ucan::did::key::Verifier::EdDsa(
+            server_sk.verifying_key(),
+        ));
 
-        let server = &EdDidKey::generate();
-        let device = &EdDidKey::generate();
+        let device = ucan::did::preset::Verifier::Key(ucan::did::key::Verifier::EdDsa(
+            ed25519_dalek::SigningKey::generate(&mut thread_rng()).verifying_key(),
+        ));
+
+        // FIXME perhaps add this back upstream as a named const
+        let varsig_header = ucan::crypto::varsig::header::EdDsaHeader {
+            codec: libipld::cbor::DagCborCodec,
+        };
+
+
+        // 1.               account -*-> server
+        // 2.                            server -a-> device
+        // 3.  dnslink -d-> account
+        // 4. [dnslink -d-> account -*-> server -a-> device]
 
         // Both of these UCANs just create ephemeral DIDs & delegate all of those
         // DID's rights to the server
-        let (account_did, account_ucan) = server_create_resource(server)?;
-        let (dnslink_did, dnslink_ucan) = server_create_resource(server)?;
+        let (account_did, _account_ucan) = server_create_resource(&server_ed_did_key)?;
+        let (dnslink_did, _dnslink_ucan) = server_create_resource(&server_ed_did_key)?;
+        let mut seed = vec![];
+
+        // 1.               account -*-> server
+        let account_pbox = ucan::delegation::Payload::powerbox(
+            account_did,
+            server,
+            "/".into(),
+            ucan::time::Timestamp::five_years_from_now(),
+        );
 
         // This UCAN gives account access to the device
-        let device_ucan: Ucan = UcanBuilder::default()
-            .for_audience(device)
-            .claiming_capability(Capability::new(
-                UcanResource::AllProvable, // UcanResource::OwnedBy(account_did.to_string()),
-                TopAbility,
-                EmptyCaveat,
-            ))
-            .witnessed_by(&account_ucan, None)
-            .sign(server)?;
+        // let device_ucan: Ucan = UcanBuilder::default()
+        //     .for_audience(device)
+        //     .claiming_capability(Capability::new(
+        //         UcanResource::AllProvable, // UcanResource::OwnedBy(account_did.to_string()),
+        //         TopAbility,
+        //         EmptyCaveat,
+        //     ))
+        //     .witnessed_by(&account_ucan, None)
+        //     .sign(server)?;
+
+        // 2.                            server -a-> device
+        let account_device_ucan = ucan::delegation::Payload {
+            // subject: None, // Some(account_did), // FIXME
+            subject: Some(account_did), // FIXME
```

Comment:
(Note that this is adapted to the more complex powerline scenario mentioned in the comment at the top of the module.)

Why the `FIXME`? This grants access to do anything (`"/"`) with the account, _not a powerline_.